### PR TITLE
chore: Switch to gtm_barebones

### PR DESCRIPTION
**Pull request by attiks**
Replace google_tag with gtm_barebones

Refs: [OPS-10039](https://humanitarian.atlassian.net/browse/OPS-10039)

[OPS-10039]: https://humanitarian.atlassian.net/browse/OPS-10039?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lazysoundsystem** [comment]
Tests failing on hr_paragraphs (?) but there's also a config file for the google_tag sensor added, not gtm_barebones

**Comment by attiks** [comment]
Fixed, there's a custom module to inject dataLayer

File: config/monitoring.sensor_config.ocha_google_tag.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+uuid: null
```
Comment (lazysoundsystem) [inline config/monitoring.sensor_config.ocha_google_tag.yml:1]: This should be the gtm_barebones sensor

Comment (attiks) [reply config/monitoring.sensor_config.ocha_google_tag.yml:1]: See https://github.com/UN-OCHA/ocha_monitoring/pull/4

**Review by lazysoundsystem — APPROVED** [review]

**Pull request merged**